src/main/resources/python/generator/gen_types.py

```
@@ -0,0 +1,180 @@
+        has_manual_iter = False
+        if name in self.context.config.manual_injections:
+            for line in self.context.config.manual_injections[name]:
+                if "__iter__" in line:
+                    has_manual_iter = True
+                    break
+
+        if has_manual_iter:
+            return []
+
+        return [
+            f"    def __contains__(self, key: Union[str, int]) -> bool: ...",
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)